esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -998,6 +999,7 @@ def get_student_schedules(request, students, prog, extra='', onsite=False):
             student.amount_finaid = iac.amount_finaid()
             student.amount_siblingdiscount = iac.amount_siblingdiscount()
             student.itemizedcosttotal = iac.amount_due()
+            student.splashinfo = SplashInfo.getForUser(student, prog)
```

Comment:
It looks like this will create a SplashInfo for the student, which we probably want to avoid when SplashInfo isn't used.

Reply:
We don't use the splash info module anymore, so it'd be fine to remove it.